### PR TITLE
feat(CLI): add debug command

### LINKED ISSUE #21619: CLI command to output engine download cache directory
Related https://github.com/prisma/prisma/issues/21616

Could be `prisma debug cache` and just output the current cache dir.

### PR DESCRIPTION
**Pull request by Jolg42**
Closes https://github.com/prisma/prisma/issues/21619

Example view from my terminal (slightly outdated already)
Screenshot 2023-10-26 at 12 21 08

### CODE REVIEW

**Comment by Jolg42** [comment]
Not ready to be merged, but review/feedback is welcome.

File: packages/cli/src/DebugInfo.ts [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    return `${underline('-- Prisma schema --')}
+Path: ${schemaPath}
+
+${underline('-- Local cache directory for engines files --')}
+${rootCacheDir}
+
+${underline('-- Environment variables --')}
+When not set, the line is dimmed and no value is displayed.
+When set, the line is bold and the value is inside the \`\` backticks.
+For more information, see https://www.prisma.io/docs/reference/api-reference/environment-variables-reference
```
Comment (janpio) [inline packages/cli/src/DebugInfo.ts:68]: Might be worth to split the Prisma specific ones from the more general ones? The link here only applied to the specific ones.

(I think the "For general debugging" and "For proxy settings" would make it into the "general" section, everything else seems Prisma specific)

**Review by janpio — APPROVED** [review]
Love it <3

File: packages/cli/src/DebugInfo.ts (hunk 1 above)
Comment (Jolg42) [reply packages/cli/src/DebugInfo.ts]: done!

**Review by millsp — APPROVED** [review]

**Pull request merged**